notebooks/covid_meds/covid_meds.Rmd

```diff
@@ -468,7 +470,7 @@ temp_ggplot <- df_scmd_ddd_map %>%
   select(year_month, ods_code, vtmnm, volume_ddd, comm_region_name) %>% 
   mutate(comm_region_name = fct_explicit_na(comm_region_name)) %>% 
   group_by(comm_region_name, vtmnm) %>%
-  summarise(volume_ddd = sum(volume_ddd)) %>%
+  summarise(volume_ddd = sum(volume_ddd, na.rm = TRUE)) %>%
   group_by(comm_region_name) %>%
   mutate(prop_use = volume_ddd / sum(volume_ddd),
```

Comment:
would also change this to `sum(volume_ddd, na.rm = TRUE)` or was there a particular reasons not to?

Reply:
Probably just missed this one, will fix, thanks!